.github/workflows/windows.yml

```diff
@@ -28,7 +28,13 @@ jobs:
           python-version: ${{ matrix.python-version }}
 
       - name: Clone gl-ci-helpers
-        run: git clone --depth 1 git://github.com/vtkiorg/gl-ci-helpers.git
+        env:
+          GIT_CURL_VERBOSE: 1
+          GIT_TRACE: 1
+          GIT_TRACE_PACKET: 1
+        run: >
+            git clone --depth 1 git://github.com/vtkiorg/gl-ci-helpers.git ||
+            git clone --depth 1 git://github.com/vtkiorg/gl-ci-helpers.git
```

Comment:
It makes sense that these two lines clone the same thing right?

Reply:
Yes, the `||` is the important bit. GHA doesn't currently have "retry" functionality.